Cargo.toml

```diff
@@ -3,11 +3,11 @@ resolver = "2"
 members = ["lgn-auth", "lgn-messages", "lgn-provers", "lgn-worker"]
 
 [workspace.dependencies]
-groth16_framework = { git = "https://github.com/Lagrange-Labs/mapreduce-plonky2.git", branch = "hack-serde" }
```

Comment:
this should have been updated before merging. I forgot to do that. The branch no longer exists.